changelogs/fragments/67508-meta-task-tags.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
I guess this might need `breaking_changes` (and a porting guide entry), since when running with `-t`, we go from always running meta tasks to now skipping them if they don't have the right tag.

Reply:
Hm... I guess so. We only documented recently that meta tasks always ran (I was originally considering it a bug rather than a feature).

Reply:
Yeah, makes sense. I just worry about people who might have automated playbook runs while depending on that previous behavior.

Reply:
Good idea, done